source/manual/schemas.html.md

```diff
@@ -0,0 +1,41 @@
+# Schemas used on GOV.UK
+Here are a list of schema's being used
+
+### [HowTo][how-to]
```

Comment:
You need to use a second-level heading here (## rather than ###) otherwise you'll get a linter error.

Reply:
This also applies to the other headings on this page